external/hbase/src/main/scala/io/gearpump/external/hbase/HBaseSink.scala

```diff
@@ -28,20 +28,13 @@ import org.apache.hadoop.hbase.{TableName, HBaseConfiguration}
 import org.apache.hadoop.hbase.client.{ConnectionFactory, Put}
 import org.apache.hadoop.hbase.util.Bytes
 
-class HBaseSink(userconfig: UserConfig, tableName: String, @transient var configuration: Configuration) extends DataSink{
+class HBaseSink(userconfig: UserConfig, tableName: String) extends DataSink{
+  @transient lazy val configuration = HBaseConfiguration.create()
```

Comment:
do we need `configuration` and `connection` elsewhere ? Why not only use them to get `table` in `open`